### PR TITLE
Fix #7566 Use the same expand arrow icon as Inspector, Console etc.

### LINKED ISSUE #7566: Use the same expand arrow icon as Inspector, Console etc.
We updated the expand arrow icon recently ([ux issue](https://github.com/devtools-html/ux/issues/31)) and implemented the new icon in mozilla-central but not yet in Debugger/Reps.

The correct icon can be found here:
https://searchfox.org/mozilla-central/source/devtools/client/themes/images/arrow.svg

It's a 10x10 icon, but is visually just 8x8. It will probably appear a little bit smaller than the current icon in Debugger.

When it's implemented as an icon-only button (e.g. if the user needs to click the icon itself, rather than the surrounding label or row), we should aim for a minimum button size of 14x14 pixels (which is tiny, but better than 10x10, and a bigger button often doesn't work with our small text and line-height).

### PR DESCRIPTION
**Pull request by ash-tamraz**
Fixes #7566 

### Summary of Changes

* Implemented new `arrow.svg` icon found in mozilla-central
* Edited stylesheets to ensure proper sizing of new arrow icon in debugger.

### Test Plan

Examined arrow icons with Firefox's DevTools' Inspector, in the Layout panel, and ran `yarn run test:all`.

### CODE REVIEW

File: src/components/shared/Svg.css [hunk 1]
```diff
@@ -353,6 +360,30 @@ html[dir="rtl"] .arrow,
   visibility: hidden;
 }
 
+.webpack svg {
```
Comment (darkwing) [inline src/components/shared/Svg.css]: Were these changes intended?

Comment (ash-tamraz) [reply src/components/shared/Svg.css]: It looks like I made a mistake with the mergetool. I've (hopefully) fixed that and I cleaned up the commits. Let me know if there are any other problems and I'll get them fixed!

**Comment by darkwing** [comment]
@nchevobbe This looks good in debugger; could you take a quick look at console?  If so we can merge.

Nice job @ash-tamraz !

**Comment by nchevobbe** [comment]
Looks fine in the console 👍
